chapters/appendix-VIII-SPDX-Lite.md

```diff
@@ -0,0 +1,52 @@
+# Appendix-VIII SPDX Lite
+
+## 1. Explanation of SPDX Lite 
+
+The SPDX Lite defines the subset of the SPDX specification, from the point of view of use cases in industries. The SPDX Lite aims at the balance between the SPDX standard and actual workflows in industries. 
```

Comment:
Thank you for proposing this addition!

Looking at the intro language in sections 1 and 2 of this appendix, I might suggest removing references to SPDX Lite being intended for use cases in industries. SPDX as a whole is intended for software bill of material use cases across industries, and I'd tend to avoid suggestions that only the parts in SPDX Lite are useful for that.

Perhaps something more like: "The term 'SPDX Lite' is meant as a shorthand for the limited set of mandatory SPDX fields, together with a couple of additional optional fields that are commonly used in many license compliance workflows."